src/components/DashboardHome/MainContent.jsx

```diff
@@ -1,58 +1,64 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import MediaQuery from 'react-responsive';
-import { breakpoints } from '@edx/paragon';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { breakpoints, StatusAlert } from '@edx/paragon';
 
 import CourseSection from '../CourseSection/CourseSection';
 import InProgressCourseCard from '../CourseCard/InProgressCourseCard';
 import UpcomingCourseCard from '../CourseCard/UpcomingCourseCard';
 import CompletedCourseCard from '../CourseCard/CompletedCourseCard';
 import Sidebar from './Sidebar';
 
-import sampleApiResponse from './sampleApiResponse';
-
 class MainContent extends Component {
-  state = {
-    courses: {
-      'in-progress': [],
-      upcoming: [],
-      completed: [],
-    },
-    // Temporarily adds `isProgramEnrollmentsLoading` to state to simulate loading
-    // state. Once we are actually calling the API and have Redux set up, this should
-    // come from props instead of state.
-    isProgramEnrollmentsLoading: true,
-  };
-
   componentDidMount() {
-    this.groupCourseEnrollmentsByStatus();
+    const programUUID = 'test-program-uuid';
```

Comment:
@adamstankiewicz I saw that [ProgramCourseEnrollmentOverviewView](https://github.com/edx/edx-platform/pull/20605/files#diff-226630538144d20a713a571c6ffe2cfbR822) requires `program_uuid` attribute. Do you know where this value would come from?

Reply:
This would be coming from gatsby-config.js and accessed through graphQL.